src/Modules/Backend/Backend/Actions/ModuleSettings.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\Backend\Actions;
+
+use ForkCMS\Core\Domain\Header\FlashMessage\FlashMessage;
+use ForkCMS\Core\Domain\Kernel\Command\ClearContainerCache;
+use ForkCMS\Modules\Backend\Domain\Action\AbstractFormActionController;
+use ForkCMS\Modules\Backend\Domain\ModuleSettings\ModuleSettingsType;
+use ForkCMS\Modules\Extensions\Domain\Module\Command\ChangeModuleSettings;
+use ForkCMS\Modules\Extensions\Domain\Module\Module;
+use ForkCMS\Modules\Extensions\Domain\Module\ModuleName;
+use RuntimeException;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ModuleSettings extends AbstractFormActionController
+{
+    protected function addBreadcrumbForRequest(Request $request): void
+    {
+        // no action specific breadcrumb needed
+    }
+
+    protected function getFormResponse(Request $request): ?Response
+    {
+        $moduleRepository = $this->getRepository(Module::class);
+        $moduleName = $this->getModuleName();
+
+        return $this->handleForm(
```

Comment:
Question: what happens if 2FA is disabled, but a user has still 2FA configured? Would it make sense that if 2FA is disabled that related settings (backup codes, trusted devices, ... ) are deleted from the users?